src/Parser/FrostMage/CombatLogParser.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+import SuggestionsTab from 'Main/SuggestionsTab';
+import Tab from 'Main/Tab';
+import Talents from 'Main/Talents';
+
+import CoreCombatLogParser from 'Parser/Core/CombatLogParser';
+import DamageDone from 'Parser/Core/Modules/DamageDone';
+
+import CastEfficiency from './Modules/Features/CastEfficiency';
+import AlwaysBeCasting from './Modules/Features/AlwaysBeCasting';
+import CooldownTracker from './Modules/Features/CooldownTracker';
+import WintersChillTracker from './Modules/Features/WintersChill';
+import BrainFreeze from './Modules/Features/BrainFreeze';
+import FingersFrost from './Modules/Features/FingersFrost';
+import IcyVeinsDuration from './Modules/Features/IcyVeinsDuration';
+import IcicleTracker from './Modules/Features/IcicleTracker';
+
+import Tier20_2set from './Modules/Items/Tier20_2set';
+import ShardofExodar from './Modules/Items/ShardofExodar';
+
+
+class CombatLogParser extends CoreCombatLogParser {
+  static specModules = {
+    // Features
+    castEfficiency: CastEfficiency,
+    alwaysBeCasting: AlwaysBeCasting,
+    cooldownTracker: CooldownTracker,
+	WintersChillTracker: WintersChillTracker,
```

Comment:
Please be consistent with existing naming conventions: module names should start with a lower case.